deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,76 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+* {
+    box-sizing: border-box;
+}
```

Comment:
de lichte rand rond de nav-balk viel me op, mogelijk komt het omdat je css reset geen margin en padding reset. Je zou die hier in de reset kunnen toevoegen met margin:0 en padding:0. Dan is die rand weg.